### DISCUSSION TITLE
Is it possible to filter out "Tent only" or "walk to" sites?

### DISCUSSION THREAD

**Discussion by sanchezluciano** [juftin/camply#75] [Q&A]
I have a campervan and I would like to see if there is a way to filter out notifications for camp sites that come up as "TENT ONLY" or "WALK TO". I see that when I get notifications there is a field specific for the type of camping (Campsite Type) so I figured it would not be to difficult to be able to filter based on that value as an improvement. 

I am super happy with this app, thank you so much for building something like this!

**Comment by juftin** [comment]
Hey @sanchezluciano thanks very much! Unfortunately that functionality isn't currently implemented in camply. However, it would absolutely be possible. The remaining work is to dig into the metadata across a bunch of campsites to see what attributes are universal for searching and what attributes are campground/campsite specific. 

More discussion is happening here: https://github.com/juftin/camply/discussions/63
